python/example_code/comprehend/TopicModeling.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+# snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+# snippet-sourcedescription:[TopicModeling.py demonstrates how to detect the topics in a document collection, check the status of the topic detection, and to show a list of all jobs submitted for the account.]
+# snippet-keyword:[Python]
+# snippet-keyword:[AWS SDK for Python (Boto3)]
+# snippet-keyword:[Code Sample]
+# snippet-keyword:[Amazon Comprehend]
+# snippet-keyword:[TopicModeling]
+# snippet-keyword:[topic modeling]
+# snippet-service:[comprehend]
+# snippet-sourcetype:[full-example]
+# snippet-sourcedate:[2019-01-31]
+# snippet-sourceauthor:[ (AWS)]
+# snippet-start:[comprehend.python.TopicModeling.complete]
+
+import boto3
+import json
+from bson import json_util
+ 
+comprehend = boto3.client(service_name='comprehend', region_name='region')
+                
+input_s3_url = "s3://input bucket/input path"
+input_doc_format = "ONE_DOC_PER_FILE"
+output_s3_url = "s3://output bucket/output path"
+data_access_role_arn = "arn:aws:iam::account ID:role/data access role"
+number_of_topics = 10
+ 
+input_data_config = {"S3Uri": input_s3_url, "InputFormat": input_doc_format}
+output_data_config = {"S3Uri": output_s3_url}
+ 
+start_topics_detection_job_result = comprehend.start_topics_detection_job(NumberOfTopics=number_of_topics,
+                                                                              InputDataConfig=input_data_config,
+                                                                              OutputDataConfig=output_data_config,
+                                                                              DataAccessRoleArn=data_access_role_arn)
+ 
+print('start_topics_detection_job_result: ' + json.dumps(start_topics_detection_job_result))
+ 
+job_id = start_topics_detection_job_result["JobId"]
+ 
+print('job_id: ' + job_id)
+ 
+describe_topics_detection_job_result = comprehend.describe_topics_detection_job(JobId=job_id)
+ 
+print('describe_topics_detection_job_result: ' + json.dumps(describe_topics_detection_job_result, default=json_util.default))

```

Comment:
Valid `bson.json_util.default` settings are `DEFAULT_JSON_OPTIONS` or `STRICT_JSON_OPTIONS`. Plus, those settings are applicable only to the `dumps` method included in the `json_util` package, but this statement uses `json.dumps`.
```suggestion
print('describe_topics_detection_job_result: ' + json.dumps(describe_topics_detection_job_result))
```